src/js/utils/strings.js

```diff
@@ -57,7 +57,7 @@ define([
             return azureFormat;
         }
 
-        path = path.substring(path.lastIndexOf('/') + 1, path.length).split('?')[0].split('#')[0];
+        path = path.substring(path.lastIndexOf('.')).split('?')[0].split('#')[0];
```

Comment:
What if there's a '.' in the search or hash portion of the url? `lastIndexOf('.')` should not be performed before splitting off the search `?` and hash `#`.